docs/source/user_guide/xmlfiles.rst

```diff
@@ -0,0 +1,154 @@
+.. _user_guide_xmlfiles:
+
+=============
+DPF XML Files
+=============
+This section describes the XML files associated with DataProcessingCore 
+and DPF plugins. These DPF files work on both Windows and Linux 
+operating systems. The files can contain content for both operating systems. 
+
+The XML files must be located alongside the plugin DLL files on Windows, 
+or SO files on Linux. 
+
+DataProcessingCore File
+-----------------------
+The content and format of the DataProcessingCore.xml file is as follows:
+
+.. code-block:: html
+
+	<?xml version="1.0"?> 
+	<DPF version="1.0"> 
+		<Environment> 
+			<Linux> 
+				<ANSYS_ROOT_FOLDER>/usr/local/ansys_inc/v222</ANSYS_ROOT_FOLDER> 
+			</Linux> 
+			<Windows> 
+				<ANSYS_ROOT_FOLDER>E:\ANSYSDev\ANSYS Inc\v222</ANSYS_ROOT_FOLDER> 
+			</Windows> 
+		</Environment> 
+		<LoadOperators> 
+			<Linux> 
+				<native> 
+					<Path>libAns.Dpf.Native.so</Path> 
+					<Loader>LoadOperators</Loader>	 
+		<UsePluginXml>false</UsePluginXml> 
+				</native> 
+			</Linux> 
+			<Windows> 
+				<Debug> 
+					<native> 
+						<Path>$(THIS_XML_FOLDER)\Ans.Dpf.NativeD.dll</Path> 
+						<Loader>LoadOperators</Loader> 
+					</native> 
+				</Debug> 
+				<Release> 
+					<native> 
+						<Path>Ans.Dpf.Native.dll</Path> 
+						<Loader>LoadOperators</Loader> 
+					</native> 
+					<fem_utils> 
+						<Path>$(THIS_XML_FOLDER)\Ans.Dpf.FEMUtils.dll</Path> 
+						<Loader>LoadOperators</Loader> 
+			<UsePluginXml>true</UsePluginXml> 
+					</fem_utils> 
+				</Release> 
+			</Windows> 
+		</LoadOperators> 
+	</DPF> 	
+
+The DataProcessingCore.xml file is provided with the DPF software. 
+Modify the file carefully to ensure that the DPF software operates correctly. 
+
+Some of the sections in the file are optional, and many of the sections 
+have Windows and Linux specific subsections. 
+
+The ``<Environment>`` section is used only for defining the ROOT folder 
+of the Ansys software. This is done with an ``<ANSYS_ROOT_FOLDER>`` tag. 
+The root folder of the Ansys software ends with the v### folder. 
+It could be something like ``C:\ansys_inc\v222``. The ANSYS_ROOT_FOLDER tag 
+defines a variable like an environment variable that can be used in the other 
+XML files. You might use it to find required third party software. 
+
+If the ANSYS_ROOT_FOLDER tag is not defined within the DataProcessing.xml file,
+the root folder is determined by either reading the AWP_ROOT### environment 
```

Comment:
Does "either" always require another possibility? That is what I personally usually understand (Sorry not my first language)

Reply:
You are right, fixed!